nrf_rpc/nrf_rpc.c

```diff
@@ -10,9 +10,9 @@
 #include <stdbool.h>
 #include <string.h>
 
-#include "nrf_rpc.h"
-#include "nrf_rpc_tr.h"
-#include "nrf_rpc_os.h"
+#include <nrf_rpc.h>
+#include <nrf_rpc_tr.h>
+#include <nrf_rpc_os.h>
```

Comment:
Don't change this.

Reply:
I tried to make it consistent with nrt_rpc.h